.eslintrc

```diff
@@ -36,8 +36,7 @@
     // The beauty of TS is that it infers types quite well, so let's not write
     // too much code.
     "@typescript-eslint/explicit-function-return-type": "off",
-    // TODO: re-enable this rule once https://github.com/mozilla/addons-code-manager/issues/85 lands.
-    "amo/only-log-strings": "off",
```

Comment:
I take it that re-enabling this means removing it from this file, as opposed to changing the value `off` to something different (like `error`)?

Reply:
yep, it is enabled by default in `eslint-plugin-amo`.